src/lib/plugin.js

```diff
@@ -189,6 +193,20 @@ class FiveBellsLedger extends EventEmitter2 {
     })
   }
 
+  _setupHttp2 (ledgerContext) {
+    const ctx = ledgerContext
+    const hostURL = parseURL(ctx.host)
+    if (hostURL.protocol === 'https:' && /^five-bells@2\d+$/.test(ctx.ledgerMetadata.version)) {
```

Comment:
I think it's better to parse out the version number and compare it to `20` instead of checking whether the version starts with a `2`.